### PR TITLE
Cargotechs now spawn with Export Scanners

### PR DESCRIPTION
**Pull request by MrDoomBringer**
**suggested by @ExcessiveUseOfCobblestone** 
Cargotechs now spawn with an export scanner in their pockets.

:cl: 
add: All Cargo Techs now come equipped with a standard supply Export Scanner! Get to selling! 
/:cl:

QM usually never bothers using the export scanner, there's literally no reason for cargotechs not to have access to one when their main job is buying and selling shit. The item is essentially useless right now, as no tech is going to bother asking the QM to haul ass to cargo to scan something. Instead, the tech will likely just sell the thing without checking price, which results in a lot of shit being sold for no reason.

### CODE REVIEW

**Comment by praisenarsie** [comment]
wtf is that diff lmao

File: code/modules/jobs/job_types/cargo_service.dm [hunk 1]
```diff
@@ -1,282 +1,281 @@
...
+	minimal_access = list(ACCESS_MAINT_TUNNELS, ACCESS_CARGO, ACCESS_CARGO_BOT, ACCESS_MAILSORTING, ACCESS_MINERAL_STOREROOM)
+
+/datum/outfit/job/cargo_tech
+	name = "Cargo Technician"
+	jobtype = /datum/job/cargo_tech
+
+	belt = /obj/item/device/pda/cargo
+	ears = /obj/item/device/radio/headset/headset_cargo
+	uniform = /obj/item/clothing/under/rank/cargotech
+	r_pocket = /obj/item/device/export_scanner
```
Comment (praisenarsie) [inline code/modules/jobs/job_types/cargo_service.dm]: spawn it in their hands

Comment (MrDoomBringer) [reply code/modules/jobs/job_types/cargo_service.dm]: k

**Comment by MrDoomBringer** [comment]
@praisenarsie diff is better now :>

**Comment by praisenarsie** [comment]
actually putting it in hands was pointless if i really think about it but eh who cares

**Comment by Iamgoofball** [comment]
I'm thinking we need to bring back the sell-a-porter

**Comment by kevinz000** [comment]
@Iamgoofball FUCK NO

**Comment by ExcessiveUseOfCobblestone** [comment]
@Iamgoofball please make it a QM traitor item for 14tc(+6 for the emag wink wink).

**Pull request merged**